docs/evm/using.mdx

```diff
@@ -34,10 +34,10 @@ Manual method: Add EVM on Flow as a custom network to MetaMask:
 | --------------- | ------------------------------------ |
 | Network Name    | Flow Testnet                         |
 | Description     | The public RPC url for Flow Testnet  |
-| RPC Endpoint    | https://testnet.evm.nodes.onflow.org |
-| Chain ID        | 545                                  |
+| RPC Endpoint    | https://mainnet.evm.nodes.onflow.org |
```

Comment:
this should be testnet url? Network Name is "Flow Testnet"

Reply:
oh I see, I guess you just need to change Network Name to Mainnet

Reply:
I think it's just supposed to be "Flow" Can we confirm with Jerome

Reply:
I missed updating the name, I thought we would want to point ppl to mainnet, now that its live